lms/djangoapps/courseware/views.py

```diff
@@ -423,6 +437,11 @@ def _index_bulk_op(request, course_key, chapter, section, position):
 
         context['show_chat'] = show_chat
 
+        if settings.FEATURES.get('ENTRANCE_EXAMS', False) and getattr(course, 'entrance_exam_enabled', False):
+            _, exam_current_score = get_entrance_exam_content_info(request, course_module, course)
+            if exam_current_score is not None:
+                context['entrance_exam_current_score'] = exam_current_score
+
```

Comment:
Question for @marcotuts, if I'm reading this implementation correctly, this will be that even when I completed the entrance exam, I will still see a a message about my exam_score when I'm navigating a course post-exam, is that desired behavior?

Otherwise, I'd suggestion checking to see if the milestone has been fulfilled, and then - if not - then we give the user feedback on his/her exam score.

Also, getting the exam_score is a bit compute intensive (not as bad as grading), so it'd be good to shut it off both the computation as well as the visual feedback.

FYI, @mattdrayer